production/helm/promtail/values.yaml

```diff
@@ -32,9 +32,9 @@ pipelineStages:
 ## Pod Labels
 podLabels: {}
 
-podAnnotations: {}
-#  prometheus.io/scrape: "true"
-#  prometheus.io/port: "http-metrics"
+podAnnotations: 
+ prometheus.io/scrape: "true"
+ prometheus.io/port: "http-metrics"
```

Comment:
Thought you were going to do this for loki as well?

Reply:
why not.